command/node_status.go

```diff
@@ -212,6 +231,14 @@ func (c *NodeStatusCommand) Run(args []string) int {
 		}
 		c.Ui.Output("\n==> Resource Utilization")
 		c.Ui.Output(formatList(resources))
+		if stats && hostStats != nil {
+			c.Ui.Output("\n===> Node CPU Stats")
```

Comment:
I think it does if we use `PrefixedUi`, I see that we are adding the markers ourselves in other places too.